cilium/cmd/status.go

```diff
@@ -69,7 +69,11 @@ func statusDaemon() {
 		healthLines = 0
 	}
 	if resp, err := client.Daemon.GetHealthz(nil); err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", pkg.Hint(err))
+		if brief {
+			fmt.Fprintf(os.Stderr, "%s\n", "cilium-health: daemon unreachable")
```

Comment:
Actually, this is not quite right. `cilium-health` is a different daemon. This is trying to get the health from the daemon, it should say something more like `cilium: daemon unreachable`.

Reply:
Thanks for reviewing, updated.